packages/did/src/DidDetails/FullDidDetails.utils.ts

```diff
@@ -28,10 +28,32 @@ const methodMapping: Record<string, VerificationKeyRelationship | undefined> = {
   web3Names: 'authentication',
 }
 
-export function getKeyRelationshipForExtrinsic(
-  extrinsic: Extrinsic
+export function getKeyRelationshipForExtrinsic<A extends AnyTuple = AnyTuple>(
+  call: CallBase<A>
```

Comment:
Can we still use `Extrinsic` as parameter? We normally use that for the user-facing API

Reply:
I changed it, because the inner call to the function doesn't have an `extrinsic` but a `CallBase`. I added another function to retain the expected interface.